packages/stable/src/types.ts

```diff
@@ -662,7 +664,8 @@ export interface DatapointsDeleteRange {
 
 export type DatapointsDeleteRequest =
   | (InternalId & DatapointsDeleteRange)
-  | (ExternalId & DatapointsDeleteRange);
+  | (ExternalId & DatapointsDeleteRange)
+  | (InstanceId & DatapointsDeleteRange);
```

Comment:
This is a duplicate of line 666?

Reply:
Not sure what you mean: line 666 is InternalId and 668 is InstanceId I think?